iroh-sync/src/ranger.rs

```diff
@@ -979,9 +983,9 @@ mod tests {
             rounds += 1;
             alice_to_bob.push(msg.clone());
 
-            if let Some(msg) = bob.process_message(msg, |_, _| {}).unwrap() {
```

Comment:
would be great to have a test in here, that checks that the validation function works as expected

Reply:
see latest commit